my-calculator/src/components/Calculator.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './style.css';
+class ResultIcone extends React.Component {
+    constructor(props){
+        super(props)
+    }
+    render(){
+        return <div className = "calculator-result"> <span> 0</span> </div>
+    }
+}
```

Comment:
[OPTIONAL] I think you can create new files for each component, this will result in a more organized project.